packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -424,12 +463,14 @@ export const getQueryBuilderPropertyNodeData = (
   property: AbstractProperty,
   parentNode: QueryBuilderExplorerTreeNodeData,
   rootMapping: Mapping,
+  modelCoverageAnalyticsResult: ModelCoverageAnalyticsResult,
```

Comment:
--> `mappingModelCoverageAnalysisResult`. Can you make sure we do this for other places as well? Thanks!